src/components/opening-hours-weekdays/DayCheckbox.tsx

```diff
@@ -14,11 +15,12 @@ const DayCheckbox = ({
   onChange: (checked: boolean) => void;
   checked?: boolean;
 }): JSX.Element => {
-  const id = `${namePrefix}-weekdays-${currentDay}`;
+  const id = sanitizeId(`${namePrefix}-weekdays-${currentDay}`);
```

Comment:
Should the name be stringifyBrackets or something?
And it should be used like this in here:
```js
const id =`${stringifyBrackets(namePrefix)-weekdays-${currentDay}`;
```

Or could this be getUiId or something else? And then it could take parts like this:
`getUiId([namePrefix, 'weekdays', currentDay])`

Reply:
> Nice first e2e-test 🏅 One little suggestion I was thinking we could have a testing strategy where we have a few e2e-tests and bunch of unit tests. I would stay away from complex component or integration tests for now

That sounds good. Would it mean that we could remove the complex component/integration tests in this change?

Reply:
> That sounds good. Would it mean that we could remove the complex component/integration tests in this change?
In this or in some other PR. Then we would have some tests and then it would be easier to change functionality if/when needed

Reply:
Looks like we do not have that many of those anymore.

Reply:
> Should the name be stringifyBrackets or something? And it should be used like this in here:
> 
> ```js
> const id =`${stringifyBrackets(namePrefix)-weekdays-${currentDay}`;
> ```
> 
> Or could this be getUiId or something else? And then it could take parts like this: `getUiId([namePrefix, 'weekdays', currentDay])`

Did the changes for the brackets removal. The `getUiId` feels brilliant. Was thinking that we could have another function which would return both name and id so we could get both in one call but did not yet go to that route..